presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,241 @@
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ==========
+``hive.metastore.authentication.type``             One of ``SIMPLE`` or ``SASL``. When set to ``SASL`` the Hive ``SIMPLE``
+                                                   connector will connect to the Hive metastore Thrift service
+                                                   using SASL and authenticate with Kerberos. When using the
+                                                   default value of ``SIMPLE``, Kerberos authentication is
+                                                   disabled and no other properties need to be configured.
+
+``hive.metastore.principal``                       The Kerberos principal of the Hive metastore. The Presto
+                                                   coordinator will use this to authenticate the Hive
+                                                   metastore.
+
+``hive.metastore.presto.principal``                The Kerberos principal that Presto will use when connecting
+                                                   to the Hive metastore.
+
+``hive.metastore.presto.keytab``                   The path to the keytab file that contains a key for the
+                                                   principal specified by ``hive.metastore.presto.principal``.
+
+================================================== ============================================================ ==========
+
+``SIMPLE`` authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    hive.metastore.authentication.type=SIMPLE
+
+``SIMPLE`` authentication is the default authentication mechanism for the Hive
+metastore. When using ``SIMPLE`` authentication, Presto connects to an
+unsecured Hive metastore. Kerberos is not used.
+
+``SASL`` authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    hive.metastore.authentication.type=SASL
+    hive.metastore.principal=hive/hive-metastore-host.example.com@EXAMPLE.COM
+    hive.metastore.presto.principal=presto@EXAMPLE.COM
+    hive.metastore.presto.keytab=/etc/presto/hive.keytab
+
+When using ``SASL`` authentication for the Hive metastore Thrift service,
+Presto will connect as the principal specified by the property
+``hive.metastore.presto.principal``.  Presto will authenticate this principal
+using the keytab specified by the ``hive.metastore.presto.keytab`` property,
+and will verify that the identity of the metastore matches
+``hive.metastore.principal``.
+
+Keytab files must be distributed to every node in the cluster that runs Presto.
+
+:ref:`Additional information on keytab files.<hive-security-additional-keytab>`
+
+Hive HDFS Authentication
+------------------------
+
```

Comment:
Change title to Hive HDFS Configuration? (Please see suggested change above for Hive Metastore Thrift Service.)